tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/dynamic-page-validator/dpv-validators/recommendation-intro-validator.js

```diff
@@ -0,0 +1,19 @@
+import ValidateContent from "../../../helpers/content-validators/content-validator.js";
+
+export const ValidateRecommendationIntro = (introPage, recommendationUrl) => {
+    const { slug, header, content } = introPage;
+    
+    cy.url().should("include", `${recommendationUrl}/${slug}`);
+
+    cy.get("h1.govuk-heading-xl").contains(header);
+ 
+    if (content) {
```

Comment:
I stole this from ValidatePage (because I couldn't call ValidatePage with an intro page because it's not a 'page'), should I do this refactor there too?